pkg/utils/random/string.go

```diff
@@ -0,0 +1,24 @@
+package random
+
+import (
+	"math/rand"
+	"time"
+)
+
+const charBytes = "abcdefghijklmnopqrstuvwxyz0123456789"
+
+func init() {
+	rand.Seed(time.Now().UTC().UnixNano())
+}
+
+// String returns a random alphanumerical string of length n.
+func String(n int) string {
+	if n < 0 {
```

Comment:
`if n <= 0` ?

Reply:
Sure, why not.
I was test-driving this with, among others:
```go
	assert.Equal(t, "", String(-1))
	assert.Equal(t, "", String(0))
```
and allocation was failing only for negative values iirc, hence I didn't go for `n <= 0`.
But this whole utility function is gone anyway, now, so we don't even have that problem anymore 🙂